src/libs/ReactNativeLauncher.js

```diff
@@ -339,13 +340,22 @@ class ReactNativeLauncher extends Launcher {
 
       const { account, trigger, job, manifest } = this.getStartContext()
       const { sourceAccountIdentifier } = this.getUserData()
-
+      const cozyFlags = listFlags()
+      const flagsWithValues = {}
+      if (cozyFlags) {
```

Comment:
Does listFlag sometimes return empty values ?

Reply:
I assumed there will always be at least one flag on every instance, but maybe it is possible yes, i can't tell right now